build-artifacts/project-template-gradle/src/main/java/com/tns/ErrorReport.java

```diff
@@ -130,64 +187,173 @@ static Intent getIntent(Context context)
 		return intent;
 	}
 
-	static boolean hasIntent(Intent intent)
-	{
+	static boolean hasIntent(Intent intent) {
 		int value = intent.getIntExtra(EXTRA_NATIVESCRIPT_ERROR_REPORT, 0);
 
 		return value == EXTRA_ERROR_REPORT_VALUE;
 	}
 
-	void buildUI()
-	{
-		Context context = activity;
+	void buildUI(AppCompatActivity activity) {
```

Comment:
we don't need activity to be bassed here, because it's already passed in constructor and saved as static